torch/distributed/fsdp/fully_sharded_data_parallel.py

```diff
@@ -313,6 +313,13 @@ class FullyShardedDataParallel(nn.Module, _FSDPState):
             prevent too many in-flight all-gathers. This ``bool`` only affects
             the sharded strategies that schedule all-gathers. Enabling this can
             help lower the number of CUDA malloc retries.
+        ignored_parameters (Optional[Iterable[torch.nn.Parameter]]): Ignored
+            parameters will not be managed by this FSDP instance,
+            that means these parameters will not be flattened and sharded by FSDP,
+            their gradients will not be synchronized as well. With this newly added
+            argument, `ignored_modules` could be deprecated soon. For back compatiablity,
+            both `ignored_parameters` and `ignored_modules` are kept for now,
+            but only one of them could not be None.
```

Comment:
nit: Just want to clarify that this is a requirement, not a suggestion
```suggestion
            but FSDP only allows one of them to be specified as not ``None``.
```